packages/block-library/src/list/save.js

```diff
@@ -1,15 +1,18 @@
 /**
  * WordPress dependencies
  */
-import { RichText, useBlockProps } from '@wordpress/block-editor';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { ordered, values, type, reversed, start } = attributes;
+	const { ordered, type, reversed, start } = attributes;
 	const TagName = ordered ? 'ol' : 'ul';
-
 	return (
-		<TagName { ...useBlockProps.save( { type, reversed, start } ) }>
-			<RichText.Content value={ values } multiline="li" />
+		<TagName
+			reversed={ reversed }
```

Comment:
Do we need to pass `reversed` and `start` twice here?

Reply:
Oops, forgot to remove it.